contracts/FrendLend.sol

```diff
@@ -0,0 +1,131 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity 0.8.7;
+import '@openzeppelin/contracts/token/ERC721/IERC721.sol';
+import '@openzeppelin/contracts/token/ERC721/extensions/ERC721URIStorage.sol';
+import '@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol';
+import './interfaces/IBullaClaim.sol';
+import './BullaBanker.sol';
+
+struct LoanOffer {
+    uint24 interestBPS; // can be 0
+    uint40 termLength; // cannot be 0
+    uint128 loanAmount;
```

Comment:
shouldn't this be 256?

Reply:
I decided to go smaller for packing reasons. A few justifications:
- V2 contracts uses 128
- Sushiswap uses 128
- uint128.max is 340,000,000,000,000,000,000 ether